### PR TITLE
Fpwd changes

### PR DESCRIPTION
**Pull request by svgeesus**
Changes to pass pubrules (HTML validity, broken links, no s[aces in IDs, and so on).

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -8319,8 +8350,8 @@ <h3>Introduction</h3>
 <h2>Archive sites</h2>
 
 <p>This specification can be found at
-<a href="http://www.w3.org/TR/2003/REC-PNG-20031110/index.html"
-><code>http://www.w3.org/TR/2003/REC-PNG-20031110/index.html</code></a>.</p>
+<a href="http://www.w3.org/TR/2022/WD-PNG-20221025/"
```
Comment (ProgramMax) [inline index.html:8353]: Same here. This link seems to work even though it is not Oct 25th yet. Is this an estimated date?

File: index.html [hunk 2]
```diff
@@ -7,17 +7,19 @@
...
+    span.chunk, a.chunk { color: #622;}
     </style>
     <script src="https://www.w3.org/Tools/respec/respec-w3c" class="remove" defer></script>
     <script class="remove">
       var respecConfig = {
           group: "png",
-          specStatus: "ED",
+          specStatus: "FPWD",
 	  shortName: "PNG",
+    publishDate: "2022-10-25",
```
[the remark this answers is not in the document]
Comment (svgeesus) [reply index.html:18]: That is the first allowed publication date. There is a publishing moratorium all week, this week; publications are on Tues and Thurs.

File: index.html [hunk 3]
```diff
@@ -1615,9 +1615,8 @@ <h3>Sequence numbers</h3>
   omitted in these tables, for clarity).
 </p>
 
-<table class="numbered simple" summary=
-"sequence numbers, if the static image is also the first frame">
-<caption>If the static image is also the first frame</caption>
+<table class="numbered simple">
+<caption>Summary numbers, if the static image is also the first frame</caption>
```
[the remark this answers is not in the document]
Comment (svgeesus) [reply index.html:1619]: Yes

**Comment by svgeesus** [comment]
Un-drafting now before the tidy changes re-so every single line :)

**Review by ProgramMax — APPROVED** [review]
LGTM

**Pull request merged**